components/director/internal/domain/systemauth/fixtures_test.go

```diff
@@ -0,0 +1,169 @@
+package systemauth_test
+
+import (
+	"database/sql/driver"
+
+	"github.com/pkg/errors"
+
+	"github.com/kyma-incubator/compass/components/director/internal/domain/systemauth"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/kyma-incubator/compass/components/director/internal/repo"
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+
+	"github.com/DATA-DOG/go-sqlmock"
+)
+
+var (
+	testTenant           = "tenant"
+	testMarshalledSchema = "{\"Credential\":{\"Basic\":{\"Username\":\"foo\",\"Password\":\"bar\"},\"Oauth\":null},\"AdditionalHeaders\":{\"test\":[\"foo\",\"bar\"]},\"AdditionalQueryParams\":{\"test\":[\"foo\",\"bar\"]},\"RequestAuth\":{\"Csrf\":{\"TokenEndpointURL\":\"foo.url\",\"Credential\":{\"Basic\":{\"Username\":\"boo\",\"Password\":\"far\"},\"Oauth\":null},\"AdditionalHeaders\":{\"test\":[\"foo\",\"bar\"]},\"AdditionalQueryParams\":{\"test\":[\"foo\",\"bar\"]}}}}"
+	testErr              = errors.New("test error")
```

Comment:
Maybe it's worth to move it to external package. Then can be used by other tests.